fandogh_cli/fandogh.py

```diff
@@ -71,7 +80,7 @@ def show_build_logs(app, version):
         sleep(1)
 
 
-@click.command('inspect')
+@click.command('inspect', cls=FandoghCommand)
```

Comment:
what about other commands?

Reply:
I only encounter the problem in this command, but we can add `cls=FandoghCommand` to all commands to be consistent